src/objects/zcl_abapgit_object_srvd.clas.abap

```diff
@@ -13,108 +13,90 @@ CLASS zcl_abapgit_object_srvd DEFINITION PUBLIC INHERITING FROM zcl_abapgit_obje
 
   PROTECTED SECTION.
 
-
   PRIVATE SECTION.
-    METHODS:
-      clear_fields
-        CHANGING
-          cs_service_definition TYPE any,
-
-      clear_field
-        IMPORTING
-          iv_fieldname          TYPE csequence
-        CHANGING
-          cs_service_definition TYPE any.
-
-    DATA:
-      mi_persistence            TYPE REF TO if_wb_object_persist,
-      mv_service_definition_key TYPE seu_objkey,
-      mr_service_definition     TYPE REF TO data.
-
-ENDCLASS.
-
-
-
-CLASS zcl_abapgit_object_srvd IMPLEMENTATION.
-
-
-  METHOD clear_field.
-
-    FIELD-SYMBOLS: <lv_value> TYPE data.
-
-    ASSIGN COMPONENT iv_fieldname OF STRUCTURE cs_service_definition
-           TO <lv_value>.
-    ASSERT sy-subrc = 0.
-
-    CLEAR: <lv_value>.
-
-  ENDMETHOD.
-
-
-  METHOD clear_fields.
-
-    clear_field(
-      EXPORTING
-        iv_fieldname          = 'CONTENT-SOURCE'
-      CHANGING
-        cs_service_definition = cs_service_definition ).
 
-    clear_field(
-      EXPORTING
-        iv_fieldname          = 'METADATA-CREATED_AT'
+    ALIASES c_abap_version_default
+      FOR zif_abapgit_object~c_abap_version_default .
+    ALIASES c_abap_version_sap_cp
+      FOR zif_abapgit_object~c_abap_version_sap_cp .
+    ALIASES gc_step_id
+      FOR zif_abapgit_object~gc_step_id .
+    ALIASES changed_by
+      FOR zif_abapgit_object~changed_by .
+    ALIASES delete
+      FOR zif_abapgit_object~delete .
+    ALIASES deserialize
+      FOR zif_abapgit_object~deserialize .
+    ALIASES exists
+      FOR zif_abapgit_object~exists .
+    ALIASES get_comparator
+      FOR zif_abapgit_object~get_comparator .
+    ALIASES get_deserialize_steps
+      FOR zif_abapgit_object~get_deserialize_steps .
+    ALIASES is_locked
+      FOR zif_abapgit_object~is_locked .
+    ALIASES jump
+      FOR zif_abapgit_object~jump .
+    ALIASES serialize
+      FOR zif_abapgit_object~serialize .
+
+    DATA mi_persistence TYPE REF TO if_wb_object_persist .
+    DATA mv_service_definition_key TYPE seu_objkey .
+    DATA mr_service_definition TYPE REF TO data .
+    CONSTANTS mc_source_file TYPE string VALUE 'srvdsrv' ##NO_TEXT.
```

Comment:
any reason why it changed from `assrvd` (which was in line with `asddls` and `asdcls`) to this?

Reply:
The reason is quite simple: I have downported the already existing SRVD object handler based in SAP Cloud Platform ABAP Environment (aka Steampunk) -- see https://github.com/SAP-samples/abap-platform-refscen-flight/tree/Cloud-Platform/src/managed